src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -383,31 +420,8 @@
   (jdbc/with-db-transaction [transaction-connection connection]
     (do-with-auto-commit-disabled transaction-connection (partial f transaction-connection))))
 
-(defn- set-timezone!
-  "Set the timezone for the current connection."
-  [driver settings connection]
-  (let [timezone      (u/prog1 (:report-timezone settings)
-                        (assert (re-matches #"[A-Za-z\/_]+" <>)))
-        format-string (sql/set-timezone-sql driver)
-        sql           (format format-string (str \' timezone \'))]
-    (log/debug (u/format-color 'green "Setting timezone with statement: %s" sql))
-    (jdbc/db-do-prepared connection [sql])))
-
-(defn- run-query-without-timezone [driver settings connection query]
-  (do-in-transaction connection (partial run-query query)))
-
-(defn- run-query-with-timezone [driver settings connection query]
```

Comment:
Dumb question, but does this affect date extracts (which should be done with respect to the reporting timezone)?